app/controllers/api/v1/clubs_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::V1::ClubsController < ApplicationController
+  def index
+    render json: ClubSerializer.new(Club.all)
+  end
+
+  def show
+    render json: ClubSerializer.new(Club.find(params[:id]))
+  end
+
+  def create
+    club = Club.create!(club_params)
```

Comment:
just double checking - do we want to keep the 'bangs' on our methods beyond the developmental phase?